kibana-reports/public/components/report_definitions/report_trigger/report_trigger.tsx

```diff
@@ -95,11 +95,16 @@ export function ReportTrigger(props: ReportTriggerProps) {
     reportDefinitionRequest.trigger.trigger_type = e;
     if (e === 'On demand') {
       delete reportDefinitionRequest.trigger.trigger_params;
-    } 
+    }
   };
 
   const handleScheduleType = (e: React.SetStateAction<string>) => {
     setScheduleType(e);
+    if (e === 'Cron based') {
```

Comment:
When same string is used for comparison in multiple places, may be it is better to have it as named constant?

Reply:
Will change